sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/jackson/JsonNodeUtils.java

```diff
@@ -10,6 +10,9 @@
 import java.util.Set;
 
 abstract class JsonNodeUtils {
+    private JsonNodeUtils(){
+        throw new IllegalStateException("Utility class");
+    }
```

Comment:
Why can't we remove the abstract and make it final, and add a private ctor?

Reply:
Yes, we can make it final if it is not designed to be extended by sub class. And I also think this way does not make sense for utility class.